mmcv/ops/csrc/pytorch/cuda/active_rotated_filter_cuda.cu

```diff
@@ -14,7 +14,7 @@ void ActiveRotatedFilterForwardCUDAKernelLauncher(const Tensor input,
   int kW = input.size(4);
   int num_rotations = indices.size(3);
   int nEntry = num_orientations * kH * kW;
-  int output_size = output.numel();
+  int output_size = input.numel();
```

Comment:
Should we also fix the unit test of the op? https://github.com/open-mmlab/mmcv/blob/master/tests/test_ops/test_active_rotated_filter.py

Reply:
Hi, @zhouzaida 
It is really hard to design unit test that can efficiently detect out of bound array access. The fact is, even though the unit test is logically correct, the buggy code could possibly also pass the unit test as long as it does not access addresses beyond the process's memory resource border.
There are indeed inefficient solutions -- repeatedly malloc the input and ground truth and run the test several times until the ground truth is corrupted by out of bounds write, or make the unit test input large enough such that the test code can read out of the process's memory resource bound. But then the code would become unacceptably ugly. LOL
@zhouzaida @zytx121
Do you have suggestions?

Reply:
It is really hard.